components/ReplySearchItem.js

```diff
@@ -0,0 +1,203 @@
+import ExpandableText from './ExpandableText';
+import VisibilityIcon from '@material-ui/icons/Visibility';
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    display: 'block',
+    position: 'relative',
+    padding: 'var(--list-item-padding)',
+    marginBottom: 12,
+    borderRadius: 8,
+    textDecoration: 'none',
+    color: 'rgba(0, 0, 0, 0.88)',
+    background: theme.palette.common.white,
+    '&:first-child': {
+      border: 0,
+    },
+    '& a': {
+      textDecoration: 'none',
+      color: 'inherit',
+    },
+  },
+  content: {
+    // fix very very long string layout
+    lineBreak: 'anywhere',
+    minWidth: 1,
+    margin: '12px 0',
+    flex: 1,
+  },
+  divider: {
+    margin: `${theme.spacing(2)}px 0`,
+    background: theme.palette.secondary[100],
+  },
+  highlight: {
+    color: theme.palette.primary[500],
+  },
+  openModalButton: {
+    position: 'relative',
+    width: '100%',
+    border: 'none',
+    outline: 'none',
+    cursor: 'pointer',
+    background: theme.palette.secondary[100],
+    color: theme.palette.secondary[300],
+    padding: theme.spacing(2),
+    borderBottomLeftRadius: 8,
+    borderBottomRightRadius: 8,
+  },
+  view: {
+    display: 'none',
+    position: 'absolute',
+    right: theme.spacing(2),
+    '& > svg': {
+      verticalAlign: 'middle',
+      marginLeft: theme.spacing(1),
+    },
+    [theme.breakpoints.up('md')]: {
+      display: 'inline',
+    },
+  },
+  article: {
+    padding: 0,
+    borderRadius: 0,
+    '&:not(:last-child)': {
+      paddingBottom: 12,
+      marginBottom: theme.spacing(3),
+      borderBottom: `1px solid ${theme.palette.secondary[200]}`,
+    },
+  },
+}));
+
+export default function ReplySearchItem({
+  id,
+  articleReplies = [],
+  query = '',
+  ...reply
+}) {
+  const articleReply = articleReplies[0];
+  if (!articleReply) return null;
+
+  const [open, setOpen] = useState(false);
+
+  const handleClickOpen = () => {
+    setOpen(true);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  const replyCount = articleReplies.length;
+
+  const {
+    article,
+    feedbacks,
+    positiveFeedbackCount,
+    negativeFeedbackCount,
+    ownVote,
+  } = articleReply || {};
+
+  const classes = useStyles();
+
+  return (
+    <li className={classes.root}>
+      <Box p={{ xs: 2, md: 4.5 }}>
+        <ArticleInfo article={article} />
+        <div className={classes.flex}>
+          <ExpandableText className={classes.content} lineClamp={3}>
+            {nl2br(article.text)}
+          </ExpandableText>
+        </div>
+        <Divider classes={{ root: classes.divider }} />
+
+        <ReplyItem
+          key={id}
+          replyId={reply.id}
+          reply={reply}
+          feedbacks={feedbacks}
+          positiveFeedbackCount={positiveFeedbackCount}
+          negativeFeedbackCount={negativeFeedbackCount}
+          ownVote={ownVote}
+          query={query}
+        />
+      </Box>
+      <button
```

Comment:
This button should show only when `articleReplies.length` > 1. If there is only one `articleReply`, it means that the reply is only used in one aritcle, and there is no "other messages" to show.

The second item in the mockup shows the variant of `ReplySearchItem` with `articleReply.length` === 1.
![image](https://user-images.githubusercontent.com/108608/83380674-0eec7c00-a411-11ea-83c2-5a3f9cf16966.png)